apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -185,22 +187,35 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
 
   blocklyWrapper.loopHighlight = function () {}; // TODO
   blocklyWrapper.getWorkspaceCode = function () {
+    return getWorkspaceCodeHelper(0, this.getHiddenDefinitionWorkspace());
```

Comment:
I could also see us passing in `MAX_GET_CODE_RETRIES` here and decrementing it. Then `retryCount` could be optional since we'd only need to try again if it was > 0.

I think this is functionally equivalent and definitely not a blocker. Just sharing how my brain processed it a little differently.

Reply:
yep that would also work! I think I'm going to leave it as-is for now.